### PR TITLE
feat(admin): added support for new role

### PR DESCRIPTION
**Pull request by mattjoke**
* new role PERUNADMINBA
* this role is the same as PERUNADMIN, but skips MFA
* regenerated openapi
* added 'PERUNADMINBA' description field to roles in en.json

### CODE REVIEW

File: libs/perun/services/src/lib/gui-auth-resolver.service.ts [hunk 1]
```diff
@@ -89,82 +89,101 @@ export class GuiAuthResolver {
...
   isOnlySponsor(): boolean {
     return this.hasAtLeastOne(Role.SPONSOR);
   }
 
   isThisGroupAdmin(id: number): boolean {
-    return this.editableGroups.includes(id) || this.principalRoles.has(Role.PERUNADMIN);
+    return (
+      this.editableGroups.includes(id) ||
+      this.principalRoles.has(Role.PERUNADMIN) ||
+      this.principalRoles.has(Role.PERUNADMINBA)
```
Comment (xflord) [inline libs/perun/services/src/lib/gui-auth-resolver.service.ts:139]: Just a thought - how about replacing all of these checks for PERUNADMIN with the call of `isPerunAdmin()`. This way should we ever remove `PERUNADMINBA` or add another similar role, the number of necessary changes will be minimal.

**Review by xflord — APPROVED** [review]
Looks ok

File: libs/perun/services/src/lib/gui-auth-resolver.service.ts (hunk 1 above)
Comment (mattjoke) [reply libs/perun/services/src/lib/gui-auth-resolver.service.ts:139]: I replaced it as much as I could, please check, if I haven't missed anything.

Comment (xflord) [reply libs/perun/services/src/lib/gui-auth-resolver.service.ts]: I think you got all of it. Have you also considered adding some readable text for the role in `en.json`? (the roles page component uses this)

**Review by bodnara — APPROVED** [review]

**Review by sarkapalkovicova — APPROVED** [review]

**Pull request merged**